fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -278,6 +280,15 @@ protected String debugString() {
     @Override
     public void init(Analyzer analyzer) throws UserException {
         super.init(analyzer);
+        if (!Util.showHiddenColumns() && olapTable.hasDeleteSign()) {
```

Comment:
Maybe you can unify this logic into one function

Reply:
olaptable may also has other hidden columns. It is not appropriate to add such a function to each hidden column, and the writing method is not particularly complicated.